pkgs/development/ocaml-modules/lacaml/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, darwin, ocaml, findlib, dune, base, stdio, liblapack, blas }:
+
+assert stdenv.lib.versionAtLeast (stdenv.lib.getVersion ocaml) "4.05.0";
+
+stdenv.mkDerivation rec {
+  pname = "ocaml${ocaml.version}-lacaml";
+  version = "11.0.3";
+
+  src = fetchFromGitHub {
+    owner = "mmottl";
+    repo = "lacaml";
+    rev = "${version}";
+    sha256 = "1aflg07cc9ak9mg1cr0qr368c9s141glwlarl5nhalf6hhq7ibcb";
+  };
+
+  buildInputs =
+    [ ocaml findlib dune base stdio liblapack blas ] ++
+    stdenv.lib.optionals stdenv.isDarwin
+      [ darwin.apple_sdk.frameworks.Accelerate ];
+
```

Comment:
@vbgl: shouldn't liblapack and blas be listed in propagatedBuildInputs instead?

Reply:
Don’t know. Do you have a test case?

Reply:
I have a private nixpkgs with a package depending on lacaml, and I had to manually add -lblas
in order to build it, or the build would fail with "-lblas: cannot find blas library". I then read a
bit about propagatedBuildInputs, changed my local version of lacaml like this:

```
     sha256 = "1aflg07cc9ak9mg1cr0qr368c9s141glwlarl5nhalf6hhq7ibcb";
   };
 
-  buildInputs =
-    [ ocaml findlib dune base stdio liblapack blas ] ++
+  buildInputs = [ ocaml findlib dune base stdio ];
+  propagatedBuildInputs = [ liblapack blas ] ++
     stdenv.lib.optionals stdenv.isDarwin
       [ darwin.apple_sdk.frameworks.Accelerate ];
```

and it worked.
Note that my understanding of propagatedBuildInputs, esp. in the context of OCaml, is very limited.

Initially, I though the full path to the blas library would be recorded in the lacaml cmxa, but ocamlobjinfo reveals that only `-cclib -lblas` is recorded there, which prevent nix to automatically record the dependency, I guess.

Other packages (including zmq) seems to use propagatedBuildInputs, I guess for this reason.

Shall I open a PR for that change to lacaml?